packages/web/src/js/Collapse.ts

```diff
@@ -136,7 +138,9 @@ class Collapse extends BaseComponent {
         this.adjustCollapsibleChildrenHeight();
       }
       this.adjustCollapsibleElementHeight(open);
-      this.target?.classList.add(CLASSNAME_TRANSITION);
+      if (this.state.init) {
+        this.target?.classList.add(CLASSNAME_TRANSITION);
```

Comment:
I leave it here just for future us: recalculation of the height is still happening, but we are not applying the transition class.